R/apply.scaling.R

```diff
@@ -1,15 +1,14 @@
 apply.scaling <- function(data.matrices, scaling.factors) {
 
-	# data.matrices can be a single matrix or a list of matrices
-	# if the data is a single matrix then the class will be 'matrix'	
-	if (class(data.matrices)[1] == 'matrix') {
-	
+	# data.matrices can be a single matrix/data frame or a list of matrices/data frames
```

Comment:
**(style|non-blocking)**  I tried to clarify the comment but does this make sense? Can you make sure to distinguish between variables (e.g. `data.matrices`) and R objects (matrix, data frame, etc)?

```suggestion
	# `data.matrices` can be either 1. a single matrix or data frame, or 2. a list of matrices or data frames
```